gtda/plotting/_plot.py

```diff
@@ -0,0 +1,32 @@
+
+import plotly.graph_objects as gobj
+from sklearn.utils.validation import check_array
+
+
+def _plot_image(image):
```

Comment:
Is this only used in `plotting/image.py`? Why the new file `_plot.py`?

Reply:
1. I. also use it for persistence images as well. this choice is debatable, but i let `HeatKernel` be plotted with contours and `PersistenceImage` as an image.
2. Initially, I was thinking about extracting `_plot_curve` from the plotting methods for `betti_curve`, `landscape`, `silhouette`, but in the end, i left the refactoring to later

Reply:
I see. If there is potential for later generality then OK, it's hidden anyway. Why contours in `HeatKernel` and not `PersistenceImage`?

Reply:
HeatKernel: if we forget about the code, the feature is a distribution, which,i thought, should be best plot with a contour.

PersistenceImage: it is intrinsically an image (as also the name suggest)

No strong opinion - we haven't discussed that and i took some liberties.

Reply:
I'm not sure -- after all, both are essentially the results of convolutions with Gaussians, and  I was not under the impression that there was a probabilistic interpretation in `HeatKernel` which is not there in `PersistenceImage`, or vice-versa.